english/is-this-a-dream/is-this-a-dream.md

```diff
@@ -1,10 +1,14 @@
 as the sharp pain moves through your nervous system and reaches your brain, the marshmallow walls begin
 to tremble stronger and stronet and brighten up until they blow up around you in a shimmering white light
 that engulfs you...you close your eyes from the strong light and when you open them up...the beautiful 	
-view from your 20th floor in central park NY and the sexy ass of the current babe you conquered last night 
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.